api/clients/codecs/blob_test.go

```diff
@@ -0,0 +1,39 @@
+package codecs
+
+import (
+	"testing"
+
+	"github.com/Layr-Labs/eigenda/common/testutils/random"
+	"github.com/stretchr/testify/require"
+)
+
+// TestBlobConversion checks that internal blob conversion methods produce consistent results
+func TestBlobConversion(t *testing.T) {
+	testRandom := random.NewTestRandom(t)
+
+	iterations := 1000
+
+	for i := 0; i < iterations; i++ {
+		originalData := testRandom.Bytes(testRandom.Intn(1024) + 1)
+		testBlobConversionForForm(t, originalData, PolynomialFormEval)
+		testBlobConversionForForm(t, originalData, PolynomialFormCoeff)
+	}
+}
```

Comment:
fuzzing would be better here as it does a smarter search than this random iid generation of test examples.

Reply:
also we should be testing the edge cases like empty bytes

Reply:
I'm on board with fuzz testing, but when do we actually run our fuzz tests? Do they run in CI?

Reply:
Took a stab at fuzz testing in [c70fad8d](https://github.com/Layr-Labs/eigenda/pull/1120/commits/c70fad8d2cb418706ae34cc9950887d9e1d826d7) LMK what you think